app/assets/javascripts/app/users/user.js

```diff
@@ -5,36 +5,40 @@
     .module('council.users')
     .controller('EditUserCtrl', EditUserController);
 
-  function EditUserController(User, userId, DS) {
+  function EditUserController($state, User, userId) {
     var ctrl = this;
 
     User.find(userId).then(resetUser);
 
-    ctrl.changePassword = changePassword;
+    ctrl.updateUser = updateUser;
     ctrl.message = '';
     cleanPasswordFields();
 
-    function changePassword() {
-      if (ctrl.password !== ctrl.passwordConfirmation) return;
+    function updateUser() {
+      if (ctrl.password !== ctrl.passwordConfirmation)
+        return;
 
       ctrl.user.update({
+        username: ctrl.username,
```

Comment:
The problem is with the password confirmation field. We'd need to rename it to `password_confirmation` here, or to `passwordConfirmation` in the backend.